### ISSUE TITLE
Error when trying to run test in iOS: Task with path 'kspTestKotlinAndroid' not found

### ISSUE THREAD

**Issue by anar110** [Kodein-Framework/MocKMP#2]
Created a new KMM project using the KMM plugin for Android Studio
Added the plugin for MocKMP:     id("org.kodein.mock.mockmp") version "1.0.1" 
Added generated dir to commonTest: kotlin.srcDir("build/generated/ksp/androidDebugUnitTest/kotlin")
Can run the RepoTest unit test for Android now. But, when run for iOS, gives the error: 

```
Could not determine the dependencies of task ':shared:compileTestKotlinIosX64'.
> Task with path 'kspTestKotlinAndroid' not found in project ':shared'.
```

Repro project attached:
[BuildTest.zip](https://github.com/Kodein-Framework/MocKMP/files/7946463/BuildTest.zip)

**Comment by SalomonBrys** [comment]
Working on this.
I did not anticipated the Android plugin working with Debug & Release.

Adding `build/generated/ksp/androidDebugUnitTest/kotlin` to Kotlin src dirs only works until you run `testReleaseUnitTest` when generated sources are then duplicated (and the compiler fails because of redeclarations).

**Comment by SalomonBrys** [comment]
Fix published in version `1.1.0`.

I've tested it with your demo project.

Note that the line `kotlin.srcDir("build/generated/ksp/androidDebugUnitTest/kotlin")` is not needed anymore.
